VocaDbWeb/webpack.mix.js

```diff
@@ -40,21 +40,13 @@ mix
 		fix: false,
 		extensions: ['ts', 'tsx'],
 	})
+	.babelConfig({
+		plugins: ['@babel/plugin-syntax-dynamic-import'],
+	})
 
-	/*.scripts([
-		"wwwroot/Scripts/jquery-2.2.1.js",
-		"wwwroot/Scripts/bootstrap.js",
-		//"wwwroot/Scripts/jquery-ui-1.10.1.js", // doesn't work if bundled together
-		"wwwroot/Scripts/knockout-3.4.1.js",
-		"wwwroot/Scripts/knockout.punches.min.js",
-		"wwwroot/Scripts/lodash.js",
-		"wwwroot/Scripts/qTip/jquery.qtip.js",
-		"wwwroot/Scripts/marked.js"
-	], "wwwroot/bundles/shared/libs.js")
+	.extract(['highcharts'], 'highcharts')
+	.extract()

```

Comment:
Currently, JavaScript code is scattered around ts and cshtml files, which makes code splitting and dynamic imports difficult. So I decided to have one big one bundle for common dependencies (e.g. jQuery, lodash and moment) except for highcharts for now.